cmake/linux/CMakeLists.txt

```diff
@@ -2,3 +2,9 @@ INSTALL(FILES lmms.png                  DESTINATION "${DATA_DIR}/pixmaps")
 INSTALL(FILES lmms                      DESTINATION "${DATA_DIR}/menu")
 INSTALL(FILES lmms.desktop              DESTINATION "${DATA_DIR}/applications")
 INSTALL(FILES lmms.xml                  DESTINATION "${DATA_DIR}/mime/packages")
+
+SET(LINUX_DESKTOP_FILE_EXEC "env QT_X11_NO_NATIVE_MENUBAR=1 lmms %f")
+SET(MENU_FILE_ICON_XPM "/usr/share/pixmaps/lmms.xpm")
```

Comment:
The name is better, but unfortunately but `${DATA_DIR}/` is relative to our source/build directories which won't be of much value for repackaging.  We luck out by allowing the other icon to simply be `lmms` (assumes it is registered elsewhere in the system), but I'd like to know more about this menu in general, perhaps we have no easy way to configure it... or perhaps we just hard-code in the most sane spot that it `should be` installed? :confused: